app/sqaws.py

```diff
@@ -137,6 +202,44 @@ def build_instance_model(pricing: PricingData, region_name: str, instance_dict:
                     nagbot_state_tag_name=nagbot_state_tag_name)
 
 
+# Get the info about a single EBS volume
+def build_volume_model(region_name: str, volume_dict: dict) -> Volume:
+    tags = make_tags_dict(volume_dict.get('Tags', []))
+
+    volume_id = volume_dict['VolumeId']
+    state = volume_dict['State']
+    volume_type = volume_dict['VolumeType']
+    name = tags.get('Name', '')
+    platform = volume_dict.get('Platform', '')
```

Comment:
Is this a property that ebs volumes actually have? Seems like this is something instances have, but volumes wouldn't have

Reply:
Hopefully I got all of the other little things now - I decided to leave this in here just because the OS info comes from the platform variable but it could probably be removed if some logic was changed in a couple other places, which I could do